test/functional/query-builder/insert-on-conflict/query-builder-insert-on-conflict.ts

```diff
@@ -51,8 +67,25 @@ describe("query builder > insertion > on conflict", () => {
 
         await connection.manager.findOne(Post, "post#1").should.eventually.be.eql({
             id: "post#1",
-            title: "Again post"
+            title: "Again post",
+            date: new Date('06 Aug 2020 00:12:00 GMT')
+        });
+
+        await connection.createQueryBuilder()
+            .insert()
+            .into(Post)
+            .values(post2)
+            .orUpdate({
+                conflict_target: ['date'],
+                overwrite: ['title']
+            })
+            .execute();
+
+        await connection.manager.findOne(Post, "post#1").should.eventually.be.eql({
```

Comment:
Can you create a separate `it` block just for this scenario?  Instead of including it in this one.

It means we can test edge cases separately and determine with them exactly what breaks